source/app/web/statics/app.placeholder.js

```diff
@@ -56,6 +56,7 @@
       animated: false,
       //Display size
       large: set.config.display === "large",
+      columns: set.config.display === "colmuns",
```

Comment:
```suggestion
      columns: set.config.display === "columns",
```

Reply:
Thanks for checking, the typo should be fixed now